### ISSUE TITLE
Need a section in installation instructions to warn user some tools will not work without API key

### ISSUE THREAD

**Issue by tjayrush** [Great-Hill-Corporation/quickBlocks#356]
Technically we use three external APIs:

1. Infura
2. Poloniex
3. Etherscan

Infura and Poloniex do not currently need a key, so maybe we can simply ignore them for the time being (or make a small note near the end of the installation instructions). Etherscan only effects 'ethSlurp' so maybe, again, a short not at the end of the installation instructions saying something like 'The ethslurp tool will not work until you get an API key from http://etherscan.io/apis. Every time you run ethSlurp without an API key, it will ask you for one until you enter it."

In other words: we don't really have to do anything here other than make notes in the installation instructions #355

**Comment by crodnun** [comment]
Added it as a dependency, at the beginning of the installation instructions.
